javascript/javascript2/week3/functionAsVariable.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+
+const arrayFunc = [
+  function functionname() {},
+
+  function functionname2() {},
+
+  function functionname3() {}
+];
+
+let arrayIteration = arrayFunc.map((individualFunct)=>{
+  return individualFunct;
+})
+
+console.log(arrayIteration);
+
+const additionFunct = function() {
+  console.log("Hello");
+};
+
+
+function addFunc(){
+}
+
+
+additionFunct();
+addFunc();
+
+const object={
+  addSum: function addFunc(){},
+}
+  console.log(object.addSum);
```

Comment:
I dont think you have 100% understood what you should do here. inside the object you redefine a function. Can you just reuse the one already defined in line 30. Here I would also recommend to define functions as variables:

`const myFunctionName = function(){...}`